test/Interop/Cxx/namespace/Inputs/classes.h

```diff
@@ -40,4 +40,35 @@ struct BasicStruct {
 };
 } // namespace ClassesNS3
 
+namespace GlobalAliasToNS1 = ClassesNS1;
```

Comment:
I was going to suggest a test that references a namespace in another module, but I think we need to sort out SR-14214 first.